package.json

```diff
@@ -29,16 +29,12 @@
     "url": "https://github.com/enpitsuLin/astro-windicss"
   },
   "dependencies": {
-    "vite-plugin-windicss": "^1.8.7"
+    "vite-plugin-windicss": "^1.8.10"
   },
   "devDependencies": {
-    "astro": "^1.0.6",
-    "tsup": "^6.2.2",
-    "typescript": "^4.7.4",
+    "astro": "^2.0.0",
+    "tsup": "^6.5.0",
+    "typescript": "^4.9.4",
     "windicss": "^3.5.6"
-  },
-  "peerDependencies": {
```

Comment:
maybe we should keep peeDependencies field to warn user this package was used in astro